packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -82,36 +83,25 @@ export const forceSourceDeploySourcePaths = async (
     }
   }
 
-  const messages: ConflictDetectionMessages = {
-    warningMessageKey: 'conflict_detect_conflicts_during_deploy',
-    commandHint: inputs => {
-      const commands: string[] = [];
-      (inputs as string[]).forEach(input => {
-        commands.push(
-          new SfdxCommandBuilder()
-            .withArg('force:source:deploy')
-            .withFlag('--sourcepath', input)
-            .build()
-            .toString()
-        );
-      });
-      const hints = commands.join('\n  ');
-
-      return hints;
-    }
-  };
-
-  const commandlet = new SfdxCommandlet<string[]>(
-    new SfdxWorkspaceChecker(),
-    new LibraryPathsGatherer(uris),
-    new LibraryDeploySourcePathExecutor(),
-    new CompositePostconditionChecker(
-      new SourcePathChecker(),
-      new TimestampConflictChecker(false, messages)
-    )
+  const messages:
```

Comment:
Same comment about how the type isn't needed since it's inferred by `getConflictMessagesFor()`